embedded-graphics/src/fonts/text.rs

```diff
@@ -431,4 +437,47 @@ mod tests {
 
         assert_eq!(text.into_pixels().count(), 6 * 12);
     }
+
+    #[test]
+    fn transparent_char() {
+        let style = TextStyle {
+            font: Font6x8,
+            text_color: None,
+            background_color: Some(BinaryColor::On),
+        };
+
+        let mut display = MockDisplay::new();
+        Text::new(" A", Point::zero())
+            .into_styled(style)
+            .draw(&mut display)
+            .unwrap();
+
+        assert_eq!(
+            display,
+            MockDisplay::from_pattern(&[
+                "#######   ##",
+                "###### ### #",
+                "###### ### #",
+                "######     #",
+                "###### ### #",
+                "###### ### #",
+                "###### ### #",
+                "############",
+            ])
+        );
+    }
+
+    #[test]
+    fn transparent_text_has_no_size_but_has_position() {
+        let style: TextStyle<BinaryColor, _> = TextStyle {
+            font: Font6x8,
+            text_color: None,
+            background_color: None,
+        };
+
+        let styled = Text::new(" A", Point::new(7, 11)).into_styled(style);
+
+        assert_eq!(Size::new(0, 0), styled.bounding_box().size);
+        assert_eq!(Point::new(7, 11), styled.bounding_box().top_left);
```

Comment:
```suggestion
        assert_eq!(
            styled.bounding_box(),
            Rectangle::new(Point::new(7, 11), Size::zero())
        );
```

Reply:
I actually prefer testing two properties separately. I think it's much more explicit in what I expect to happen. Merging the tests makes the expected size (*the* thing I want to test) be hidden in a construct that's ... collateral.

Reply:
I would also prefer the code @jamwaffles has suggested, because it is more consistent with the other tests. The name of the test is a bit misleading, because every text has a size, it just happens to be `Size::zero()` in this case. If I would write the test I would simply call it `transparent_text_bounding_box` or `transparent_text_dimensions` (because of the `Dimenstions` trait) and add a comment that describes the expected behavior, which is often easier than trying to come up with a function name that conveys the same information.